test/data/slides.bmp

```diff
@@ -0,0 +1,18 @@
+{"Body": {"BGPUpdate": {"Body": {"PathAttributes": [{"nexthop": "10.0.2.1", "type": 3}], "NLRI": [{"prefix": "0.0.0.0/0"}], "WidthdrawnRoutes": []}}}, "Header": {"Type": 0}, "PeerHeader": {"PeerBGPID": "10.0.1.2", "Timestamp": "10"}}
```

Comment:
These IP prefixes look very different from the ones that were specified in the CVS file. Am I missing something?

Reply:
Perhaps? They look the same to me as the first line of slides.csv.

What output would you expect for this line?